coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/CoralToTrinoSqlCallConverter.java

```diff
@@ -107,6 +108,8 @@ protected SqlCall transform(SqlCall sqlCall) {
             null, null),
         new ToDateOperatorTransformer(configs.getOrDefault(AVOID_TRANSFORM_TO_DATE_UDF, false)),
         new CurrentTimestampTransformer(), new FromUnixtimeOperatorTransformer(),
+        // Avoid unintentional transformation of Trino's from_unixtime SQL calls
```

Comment:
No need for the comment since the operators are different so there is no ambiguity/confusion.

Reply:
I was more concerned about the placement of the transformer. It needs to be placed after `FromUnixtimeOperatorTransformer`